stream_alert/rule_processor/handler.py

```diff
@@ -47,7 +48,7 @@ def __init__(self, context, enable_alert_processor=True):
         """
         # Load the config. Validation occurs during load, which will
         # raise exceptions on any ConfigErrors
-        self.config = load_config()
+        self.__config = getattr(StreamAlert, '__config', load_config())
```

Comment:
I don't think this code will ever actually sets the class property of `StreamAlert.__config` and thus it will always be empty and load_config() will still get called each time regardless.

Suggestion: avoid actually setting this value on the _instance_ and do a simple ternary-esque operator check like:

`StreamAlert.__config = StreamAlert.__config or load_config()`

And then always reference the config on the class, not instance